bin/genesis

```diff
@@ -139,7 +139,13 @@ setup() {
 		while IFS='' read -r line || [[ -n "$line" ]]; do
 			cmd=$(echo $line | perl -pe 's|^(.*?):.*$|$1|')
 			ver=$(echo $line | perl -pe 's|^.*?: *(.*)$|$1|')
-			if [[ -z $(command -v ${cmd}) ]]; then
+			if [[ ${cmd} == "genesis" ]] ; then
+				if [[ ${ver} != ${VERSION} ]] ; then
```

Comment:
Pinning to a specific version may be too restrictive; do we want to support a minimum version ie (>= v1.2.3)